src/pineko/parser.py

```diff
@@ -63,9 +148,132 @@ def get_yaml_information(yaml_file, grids_folder):
         tmp = []
         for member in operand:
             p = grids_folder / f"{member}.{EXT}"
-            if not p.exists():
+            if not p.exists() and check_grid_existence:
                 raise GridFileNotFound(f"Failed to find {p}")
             tmp.append(p)
         ret.append(tmp)
 
     return yaml_content, ret
+
+
+def _pinelumi_to_columns(pine_luminosity, hadronic, flav_size=14):
```

Comment:
If you're going to move this out, maybe I'll carefully review it only on the other side.

However, I haven't tested, but it looks mostly fine, apart from:
1. `co` is a cryptic name for a variable (I understand is a column, but took me a while, so maybe you can call it `col_indices`, `column_indices`, `col_labels`, or something like...)
2. the value 14 that is a default here, it's not modifiable for the user of `pineappl_to_fktable`, resulting in an hardcoded value on this side (`pineko`), that you will have to match on the other side (`vp`); one more reason to move it...

Reply:
This can stay here though, since where the columns are is defined by eko. This one I don't know tbh.

Reply:
Then it should be accessible even from the caller, or if 14 it's really an hard-coded value, instead of putting as a default argument, make it a module level constant  (i.e. an uppercase named variable).

Reply:
I guess it's "hardcoded" but it should be hardcoded to len(whatever eko size is).